decidim-elections/app/controllers/decidim/elections/admin/answers_controller.rb

```diff
@@ -7,7 +7,13 @@ module Admin
       class AnswersController < Admin::ApplicationController
         include Decidim::Proposals::Admin::Picker
         helper Decidim::ApplicationHelper
-        helper_method :election, :question, :answers, :answers
+        helper_method :election, :question, :answers, :answers, :missing_answers
+
+        def index
+          if question.max_selections > answers.count
```

Comment:
Do you think that checking for `missing_answers.any?` would be more DRY? :)

Reply:
Thanks. Yes, totally. I'll update it!